src/views/SignDidStart/SignDidStart.tsx

```diff
@@ -0,0 +1,86 @@
+import { browser } from 'webextension-polyfill-ts';
+
+import { Link, useLocation } from 'react-router-dom';
+
+import { filter } from 'lodash-es';
+
+import { useEffect } from 'react';
+
+import * as styles from './SignDidStart.module.css';
+
+import { Identity } from '../../utilities/identities/types';
+import { IdentitiesCarousel } from '../../components/IdentitiesCarousel/IdentitiesCarousel';
+import { generatePath, paths } from '../paths';
+import { isFullDid } from '../../utilities/did/did';
+import { useIdentityCredentials } from '../../utilities/credentials/credentials';
+import { usePopupData } from '../../utilities/popups/usePopupData';
+import { SignDidOriginInput } from '../../channels/SignDidChannels/types';
+
+interface Props {
+  identity: Identity;
+  onPopupData: (popupData: SignDidOriginInput) => void;
+}
+
+export function SignDidStart({ identity, onPopupData }: Props) {
+  const t = browser.i18n.getMessage;
+
+  const { search: popupQuery } = useLocation();
+
+  const popupData = usePopupData<SignDidOriginInput>();
+  useEffect(() => {
+    onPopupData(popupData);
+  }, [popupData, onPopupData]);
+
+  const { address, did } = identity;
+
+  const credentials = useIdentityCredentials(did);
+
+  const attestedCredentials = filter(
+    credentials,
+    ({ status }) => status === 'attested' || status === 'pending',
+  );
+
+  const errorDid = !isFullDid(did) && t('view_SignDidStart_error_did');
+  const errorCredentials = [
+    errorDid,
+    !attestedCredentials?.length && t('view_SignDidStart_error_credentials'),
+  ].filter(Boolean)[0];
```

Comment:
Would this be more consistent?
```suggestion
  const errorDid = !isFullDid(did) && t('view_SignDidStart_error_did');
  const errorUnattested = !attestedCredentials?.length && t('view_SignDidStart_error_credentials')
  const errorCredentials = [errorDid, errorUnattested].filter(Boolean)[0];
```